### ISSUE TITLE
CRF is broken with Theano backend

### ISSUE THREAD

**Issue by neingeist** [farizrahman4u/keras-contrib#131]
With Theano (git version due to compiler issues) the CRF is broken. Unfortunately I am unable to test this with the current stable Theano.

~~~
% python conll2000_chunking_crf.py
Using Theano backend.
[...]
Epoch 1/10
8936/8936 [==============================] - 15s - loss: nan - viterbi_acc: 0.0098 - val_loss: nan - val_viterbi_acc: 0.0093
Epoch 2/10
8936/8936 [==============================] - 15s - loss: nan - viterbi_acc: 0.0098 - val_loss: nan - val_viterbi_acc: 0.0093
~~~

With the TensorFlow backend, the example runs perfectly on my machine.

**Comment by joelkuiper** [comment]
Can confirm, same on my machine

**Comment by neingeist** [comment]
@joelkuiper Which Theano version are you using?

**Comment by neingeist** [comment]
The problem is also there with Theano-rel-0.10.0beta1.

**Comment by FredRodrigues** [comment]
Yup ! I also confirm!

**Comment by Euruson** [comment]
I met the same problem with ``Keras 2.1.5`` and ``Theano 1.0.1`` on Win10.
![image](https://user-images.githubusercontent.com/27568823/40537596-e7b79b48-6041-11e8-8771-aeb5fd29efd7.png)

**Comment by skhong0831** [comment]
I have the same problem with keras 2.1.5 and theano 1.0.1 on ubuntu 16.
